tests/unit/test_exectype.py

```diff
@@ -28,18 +40,21 @@ def assert_bi():
     # we override BaseIndexer now, without force it shall not store all init values
```

Comment:
Would you please change the comments here as well? By this point, `BaseIndexer` has not registered in `reg_cls_set` yet and `store_init_kwargs` will be executed

Reply:
Done!